config/initializers/sucker_punch.rb

```diff
@@ -0,0 +1,2 @@
+require 'sucker_punch/async_syntax'
+SuckerPunch.logger = Logger.new("#{Rails.root}/log/sucker_punch.log")
```

Comment:
maybe i can comment row 2 and leave the .gitkeep on log folder or you prefer that i'll remove both?
The convention of sucker_punch gem is to log on rails logger as you suggested.